src/graph/charttitles/charttitles.impl.pas

```diff
@@ -91,5 +100,16 @@ function TChartTitlesConfiguration.SetY2AxisTitle(Text: UTF8String): IChartTitle
   Result := Self;
 end;
 
+function TChartTitlesConfiguration.SetStratumValue(Value: UTF8String): IChartTitleConfiguration;
+begin
+  FStratumValue := Value;
+  Result := Self;
+end;
+
+constructor TChartTitlesConfiguration.Create;
+begin
+  FStratumValue := '';
```

Comment:
I don't think this initialization is nessesary, since all strings are initialized to '' (empty) during create.

If you want to keep this, at least all the other member variables with UTF8String should be initialized to '' here as well.

Reply:
removed